docs/DeveloperGuide.md

```diff
@@ -944,6 +944,138 @@ testers are expected to do more *exploratory* testing. Alternative test cases wi
 - `remark 100000 r/Birthday on 31 July.`.
 - `remark 1 r/Birthday on 31 July r/`.
 
+### Scheduling a meeting with a person
+
+#### Scheduling a meeting with a person successfully
+##### The time input of this test must be after current date time.
+
+**Prerequisites:**
+1. List all persons using the `list` command. At least 5 persons are in the list.
+2. No person in the list has any scheduled meeting.
+
+**Example Test Case**: `schedule 1 s/23-04-2025T16:00 e/23-04-2025T17:00`.
```

Comment:
I think the note that the input start datetime and end datetime should be placed here?